index.js

```diff
@@ -1,18 +1,19 @@
-require('dotenv').config(); //initialize dotenv
+require('dotenv').config();
 const db = require('./db');
 const ai = require('./ai');
 const { Events, GatewayIntentBits } = require('discord.js');
-const Discord = require('discord.js'); //import discord.js
+const Discord = require('discord.js'); 
 
 // Get the keys
-const OPENAI_API_KEY = process.env.OPENAI_API_KEY;
-const AI_MODEL = process.env.AI_MODEL;
-const DISCORD_TOKEN = process.env.DISCORD_TOKEN;
-const AI_COMMAND_NAME = process.env.AI_COMMAND_NAME;
-const AI_CHANNEL = process.env.AI_CHANNEL;
-const CONTENT_EMOJI = process.env.CONTENT_EMOJI;
+const {
```

Comment:
Wait is this just the exact same as what I'd written, just more concise?

Reply:
Yeah, this was the same, just less duplication.